pkg/services/ngalert/database.go

```diff
@@ -119,3 +119,16 @@ func (ng *AlertNG) getAlertDefinitions(cmd *listAlertDefinitionsCommand) error {
 		return nil
 	})
 }
+
+func (ng *AlertNG) getAlertDefinitions(cmd *listAlertDefinitionsCommand) error {
```

Comment:
This is a query not a command, the idea behind the Query and Command pattern is to really make clear what actions change data want are just reads